src/applications/personalization/profile360/components/BankInfoForm.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
```

Comment:
Almost all of this was pulled out of the `PaymentInformationEditModal` component. Having this in its own component makes it easy to drop it into Profile 2.0's edit-in-place feature.